src/protocols/http/hpr_http_roaming_utils.erl

```diff
@@ -0,0 +1,106 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 17. Sep 2022 10:56 AM
+%%%-------------------------------------------------------------------
+-module(hpr_http_roaming_utils).
+-author("jonathanruttenberg").
+
+-define(RESPONSE_STREAM_ETS, hpr_http_response_stream_ets).
+
+%% API
+-export([
```

Comment:
Could we get some EUnits on those functions?

Reply:
Will do.

Reply:
Done.